framework/format/api_call_id.h

```diff
@@ -653,7 +655,519 @@ enum ApiCallId : uint32_t
+    ApiCall_IDXGIOutput3_CheckOverlaySupport                                                      = MakeApiCallId(ApiFamily_Dxgi, 0x107f),
+
+    // dxgi1_4.h
+    ApiCall_IDXGISwapChain3_GetCurrentBackBufferIndex                                             = MakeApiCallId(ApiFamily_Dxgi, 0x1080),
+    ApiCall_IDXGISwapChain3_CheckColorSpaceSupport                                                = MakeApiCallId(ApiFamily_Dxgi, 0x1081),
+    ApiCall_IDXGISwapChain3_SetColorSpace1                                                        = MakeApiCallId(ApiFamily_Dxgi, 0x1082),
+    ApiCall_IDXGISwapChain3_ResizeBuffers1                                                        = MakeApiCallId(ApiFamily_Dxgi, 0x1083),
+    ApiCall_IDXGIOutput4_CheckOverlayColorSpaceSupport                                            = MakeApiCallId(ApiFamily_Dxgi, 0x1084),
+    ApiCall_IDXGIFactory4_EnumAdapterByLuid                                                       = MakeApiCallId(ApiFamily_Dxgi, 0x1085),
+    ApiCall_IDXGIFactory4_EnumWarpAdapter                                                         = MakeApiCallId(ApiFamily_Dxgi, 0x1086),
+    ApiCall_IDXGIAdapter3_RegisterHardwareContentProtectionTeardownStatusEvent                    = MakeApiCallId(ApiFamily_Dxgi, 0x1087),
+    ApiCall_IDXGIAdapter3_UnregisterHardwareContentProtectionTeardownStatus                       = MakeApiCallId(ApiFamily_Dxgi, 0x1088),
+    ApiCall_IDXGIAdapter3_QueryVideoMemoryInfo                                                    = MakeApiCallId(ApiFamily_Dxgi, 0x1089),
+    ApiCall_IDXGIAdapter3_SetVideoMemoryReservation                                               = MakeApiCallId(ApiFamily_Dxgi, 0x108a),
+    ApiCall_IDXGIAdapter3_RegisterVideoMemoryBudgetChangeNotificationEvent                        = MakeApiCallId(ApiFamily_Dxgi, 0x108b),
+    ApiCall_IDXGIAdapter3_UnregisterVideoMemoryBudgetChangeNotification                           = MakeApiCallId(ApiFamily_Dxgi, 0x108c),
+
+    // dxgi1_5.h
+    ApiCall_IDXGIOutput5_DuplicateOutput1                                                         = MakeApiCallId(ApiFamily_Dxgi, 0x108d),
+    ApiCall_IDXGISwapChain4_SetHDRMetaData                                                        = MakeApiCallId(ApiFamily_Dxgi, 0x108e),
+    ApiCall_IDXGIDevice4_OfferResources1                                                          = MakeApiCallId(ApiFamily_Dxgi, 0x108f),
+    ApiCall_IDXGIDevice4_ReclaimResources1                                                        = MakeApiCallId(ApiFamily_Dxgi, 0x1090),
+    ApiCall_IDXGIFactory5_CheckFeatureSupport                                                     = MakeApiCallId(ApiFamily_Dxgi, 0x1091),
+
+    // dxgi1_6.h
+    ApiCall_DXGIDeclareAdapterRemovalSupport                                                      = MakeApiCallId(ApiFamily_Dxgi, 0x1092),
+    ApiCall_IDXGIAdapter4_GetDesc3                                                                = MakeApiCallId(ApiFamily_Dxgi, 0x1093),
```

Comment:
Where is the DXGI 1.6 function `DXGIDisableVBlankVirtualization` or where is its generation suppressed in the Python codegen if capturing it is not wanted?
If it should be captured, wouldn't its logical call id be `0x1093`, and shouldn't an ID close to that be reserved for it somewhere around this line at least?

https://learn.microsoft.com/en-us/windows/win32/api/dxgi1_6/nf-dxgi1_6-dxgidisablevblankvirtualization